src/Ext/freecad/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 EXECUTE_PROCESS(COMMAND ${PYTHON_EXECUTABLE} -c 
-"from distutils.sysconfig import get_python_lib; print(get_python_lib(plat_specific=True))"
```

Comment:
For me it has the same behaviour as in [src/Mod/Path/libarea/CMakeLists.txt](https://github.com/FreeCAD/FreeCAD/pull/6753/files/a839658c3d9d9cdb7e5a6071e5f4d9d643665679#diff-259df394b9555c95f6c7a3f3e0b045c48931e413e30ae9a5e659b888fedb365a)